packages/flutter_tools/test/integration.shard/flutter_attach_test.dart

```diff
@@ -197,24 +197,20 @@ void main() {
     testWithoutContext('enables Observatory on run', () async {
         await flutterRun.run(
           withDebugger: true,
-          // TODO(bkonyi): uncomment once Observatory is disabled by default
-          // See https://github.com/dart-lang/sdk/issues/50233
-          // serveObservatory: true,
+          serveObservatory: true,
         );
         expect(await isObservatoryAvailable(), true);
     });
 
     testWithoutContext('enables Observatory on attach', () async {
-      await flutterRun.run(withDebugger: true, serveObservatory: false);
+      await flutterRun.run(withDebugger: true);
       // Bail out if Observatory is still served by default in the VM.
       if (await isObservatoryAvailable()) {
         return;
       }
```

Comment:
Can we update this then to:
```suggestion
      expect(await isObservatoryAvailable(), isFalse);
```

Reply:
No, since the change in the VM hasn't rolled in yet.

Reply:
Which change?

Reply:
It was reverted in https://github.com/dart-lang/sdk/commit/5a8ddc0756684e8ebfbc0edfbfc6046755b8394a, which will reland once this change lands.